source/docs/05-mensural.xml

```diff
@@ -165,33 +165,33 @@
                      </figure> The same applies to the examples that follow.</p>
                   <p>The following example illustrates an <hi rend="italic">imperfection</hi> (the two <hi rend="italic">longae</hi>) in <hi rend="italic">modus minor perfectus</hi> with the same <hi rend="italic">longa</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">longa</hi> sequence but with an additional <hi rend="italic">dot of division</hi> between the two <hi rend="italic">breves</hi> (see <ptr target="#dots"/> for more details). Notice that here the <hi rend="italic">longae</hi> have been imperfected, unlike the previous example in which they kept the perfect value indicated by the mensuration. <figure>
-                        <head/>
+                        <head>Example on imperfection: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/imperfection.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>The following example in <hi rend="italic">modus minor imperfectus</hi> illustrates the use of a dot of augmentation following the <hi rend="italic">longa</hi> (see <ptr target="#dots"/> for more details). Notice that the <hi rend="italic">longa</hi>, which is supposed to be imperfect according to the mensuration, has a perfect value due to the augmentation dot. <figure>
-                        <head/>
+                        <head>Example on augmentation: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/augmentation.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>Finally, the following example illustrates the <att>dur.quality</att> values related to the <hi rend="italic">Ars antiqua</hi> style, for <hi rend="italic">perfect modus</hi>, with the breve equivalents notated in the lower staff for reference (as in the previous examples). <figure>
-                        <head/>
+                        <head>Example on ars antiqua related values for perfect modus: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```

Comment:
```suggestion
                        <head>Example on ars antiqua related values for perfect modus (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

Reply:
I suggest:

"Ars antiqua-related values for perfect modus."

Reply:
And so on ...